### DISCUSSION TITLE
Gitea as Git source

### DISCUSSION THREAD

**Discussion by abernh** [coollabsio/coolify#1963] [Feature Requests]
Previously asked in https://github.com/coollabsio/coolify/issues/162

This is about being able to use a Gitea server as source when creating "new git source" the same way it is currently possible to use GitHub or GitLab.

**Comment by bioshazard** [comment]
Yes please. I am obsessed with the idea of deploying Gitea in Coolify and using that deployment as a source.

**Comment by typed-sigterm** [comment]
+1 We are troubled by this too.

**Comment by TriHydera** [comment]
Yes

**Comment by AdilsonTorres** [comment]
+1

**Comment by Qubi-B** [comment]
+100

**Comment by Qubi-B** [comment]
shi, ik we are not a large group but for ppl with mostly self-hosted infra this would be so helpful...

**Comment by AbhijithGanesh** [comment]
+100
